src/Microsoft.AspNet.Mvc.Localization/ViewLocalizer.cs

```diff
@@ -65,7 +65,6 @@ public void Contextualize(ViewContext viewContext)
             {
                 baseName = baseName.Substring(1);
             }
-            baseName = _applicationName + "." + baseName;
```

Comment:
yes. This assumed resx file is always at root path. Underlying `ResourceManagerStringLocalizerFactory` now takes the basename and appends resource folder path(if any) to form full path of resx file